content/en/docs/ambient/usage/verify-mtls-enabled/index.md

```diff
@@ -0,0 +1,91 @@
+
+{{< text syntax=bash >}}
+$ istioctl ztunnel-config workloads
+NAMESPACE    POD NAME                                IP         NODE                     WAYPOINT PROTOCOL
+default      details-v1-857849f66-ft8wx              10.42.0.5  k3d-k3s-default-agent-0  None     HBONE
+default      kubernetes                              172.20.0.3                          None     TCP
+default      productpage-v1-c5b7f7dbc-hlhpd          10.42.0.8  k3d-k3s-default-agent-0  None     HBONE
+default      ratings-v1-68d5f5486b-b5sbj             10.42.0.6  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v1-7dc5fc4b46-ndrq9             10.42.1.5  k3d-k3s-default-agent-1  None     HBONE
+default      reviews-v2-6cf45d556b-4k4md             10.42.0.7  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v3-86cb7d97f8-zxzl4             10.42.1.6  k3d-k3s-default-agent-1  None     HBONE
+{{< /text >}}
+
+Having HBONE configured on your workload doesn't mean your workload will reject any plaintext traffic. If you want your workload to reject plaintext traffic, create a `PeerAuthentication` policy with mTLS mode set to `STRICT` for your workload.
+
+## Validate mTLS from metrics
+
+If you have [installed Prometheus](docs/ops/integrations/prometheus/#installation), you can setup port-forwarding and open the Prometheus UI by using the following command:
+
+{{< text syntax=bash >}}
+$ istioctl dashboard prometheus
+{{< /text >}}
+
+In Prometheus, you can view the values for the TCP metrics. First, select Graph and enter the a metric such as: `istio_tcp_connections_opened_total`, `istio_tcp_connections_closed_total`, `istio_tcp_received_bytes_total`, or `istio_tcp_sent_bytes_total`. Lastly, click Execute. The data will contain entries such as:
+
+{{< text syntax=plain >}}
+istio_tcp_connections_opened_total{
+  app="ztunnel",
+  connection_security_policy="mutual_tls",
+  destination_principal="spiffe://cluster.local/ns/default/sa/bookinfo-details",
+  destination_service="details.default.svc.cluster.local",
+  reporter="source",
+  request_protocol="tcp",
+  response_flags="-",
+  source_app="sleep",
+  source_principal="spiffe://cluster.local/ns/default/sa/sleep",source_workload_namespace="default",
+  ...}
+{{< /text >}}
+
+Validate that the `connection_security_policy` value is set to `mutual_tls` along with the expected source and destination identity information.
+
+## Validate mTLS from logs
+
+You can also view either the source or destination ztunnel log to confirm mTLS is enabled, along with peer identities. Below is an example of the source ztunnel's log for a request from the `sleep` service to the `details` service:
+
+{{< text syntax=plain >}}
+2024-08-21T15:32:05.754291Z	info access	connection complete	src.addr=10.42.0.9:33772 src.workload="sleep-7656cf8794-6lsm4" src.namespace="default"
```

Comment:
```suggestion
2024-08-21T15:32:05.754291Z    info access    connection complete    src.addr=10.42.0.9:33772 src.workload="sleep-7656cf8794-6lsm4" src.namespace="default"
```

This should fix the linter

Reply:
Thanks Daniel! Ian also pinged me, fingers crossed this is it!!

Reply:
mark this as resolved as i checked in a version fixed this around same time.